README.md

```diff
@@ -26,6 +27,7 @@ Install & Upgrade Tests |
 [![RefApp 2.x Inpatient](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-inpatient.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-inpatient.yml)
 [![RefApp 2.x Registration](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-registration.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-registration.yml)
 [![RefApp 2.x User Account](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-user-account.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-user-account.yml)
+[![RefApp 2.x Find-Patient](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-find-Patient.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-find-Patient.yml)
```

Comment:
As the .yml file is renamed the stattus buldge markdown changes. I guess to be now

` [![RefApp 2.x Find Patient](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2.x-find-Patient.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2.x-find-Patient.yml)`

Reply:
Is there a  difference than the above change

Reply:
> @kdaud i didnt get your point above , Is there a difference than the above chang


As the .yml was renamed from refapp-2x-findPatient.yml --> refapp-2x-find-patient.yml, the status badge mark down changed as well from `[! [RefApp 2.x Find-Patient].....](),` to `[! [RefApp 2.x Find Patient].....](),`

Reply:
Nice catch Resolved thanks